src/main/java/com/google/codeu/data/CohortDatastore.java

```diff
@@ -51,4 +53,14 @@ public Cohort getCohort(Entity cohortEntity) {
         return new Cohort(cohortID, cohortName);
     }
 
+    public Set<String> getCohorts(){
```

Comment:
You actually want to return a list of cohort models:
```java
public List<Cohort> getCohorts();
```
Granted the UI will only display the cohortName it will also need to render links to the list of teams for a given cohort.